plugin.yaml

```diff
@@ -128,94 +128,158 @@ data_types:
           - 10.10.0.0/16
           - 172.16.0.0/16
         required: true
+        description:
+          The address space.
+          This should contain a list of ``addressPrefixes``.
       dhcpOptions:
         required: false
+        description:
+          list of DHCP options
       subnets:
         required: false
-        
+        description:
+          list of subnets
+
   cloudify.datatypes.azure.network.NetworkSecurityGroupConfig:
     description: >
       See https://msdn.microsoft.com/en-us/library/mt163656.aspx
     properties:
       securityRules:
         required: false
-        
+        description:
+          A list of rules
+
   cloudify.datatypes.azure.network.NetworkSecurityRuleConfig:
     description: >
       See https://msdn.microsoft.com/en-us/library/mt163645.aspx
     properties:
       description:
+        description: >
+          description
         type: string
         required: false
       protocol:
+        description: TCP, UDP, or *
         type: string
         required: true
       sourcePortRange:
+        description: |
+          Single port number from 1 to 65535,
+          port range (i.e. 1-65635),
+          or * (for all ports)
```

Comment:
Let's keep the contents as consistent as we can right now. Reviewing the entire plugins section is on my agenda for the near future.